lib/utils.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const assert = require('assert');
+const crypto = require('crypto');
+const Boom = require('boom');
+const bundleJS = require('@asset-pipe/js-reader');
+const bundleCSS = require('@asset-pipe/css-reader');
+const { promisify } = require('util');
+const body = promisify(require('body/json'));
+const schemas = require('./schemas');
+
+function createHashFromContent(content) {
+    const hasher = crypto.createHash('sha256');
+    hasher.update(content, 'utf8');
+    return hasher.digest('hex');
+}
```

Comment:
This method exist in the common module: https://github.com/asset-pipe/asset-pipe-common/blob/master/lib/common.js#L103

The common module was introduced to keep all hash methods in one place to make sure we use the same hash algorithms everywhere. I think we should stick to that approach.

Reply:
Great catch!